legion/engine/core/scheduling/scheduler.cpp

```diff
@@ -43,9 +46,10 @@ namespace legion::core::scheduling
             if (instruction)
             {
                 {
-                    OPTICK_EVENT("Executing command")
```

Comment:
This frightens me, how was this a line without a semicolon, and yet no one noticed

Reply:
it's a macro, it doesn't necessarily need a semicolon in this case